content.js

```diff
@@ -45,3 +54,62 @@ function hidePopup() {
     overlay = null;
     frame = null;
 }
+
+function findCard() {
```

Comment:
Function moved from `popup.js`, see actual diff in [this commit](https://github.com/bdebon/youtube-thumbnail-tester-chrome-extension/pull/37/commits/1fb19d27d3544432af31a92d780a70a6d5bf0386)